docs/transformers/external/starlark.md

```diff
@@ -12,4 +12,76 @@ nav_order: 1
 
 A `Starlark` class based transformer allows for writing a full fledged transformer in [starlark](https://docs.bazel.build/versions/2.1.0/skylark/language.html) by implementing the `directory_detect` and `transform` functions. 
 
-Samples of use of this transform class can be found in [here](https://github.com/konveyor/move2kube-transformers/tree/main/add-custom-kubernetes-annotation) and [here](https://github.com/konveyor/move2kube-transformers/tree/main/add-custom-files-directories-in-custom-locations)
\ No newline at end of file
+Samples of use of this transform class can be found in [here](https://github.com/konveyor/move2kube-transformers/tree/main/add-custom-kubernetes-annotation) and [here](https://github.com/konveyor/move2kube-transformers/tree/main/add-custom-files-directories-in-custom-locations)
+
+## Functions
+
+### Transform
+
+* `base_directory_detect()`, `directory_detect()`, `transform()` - These functions are used for Move2Kube transformtion and details can be found [here](https://move2kube.konveyor.io/concepts/transformer#methods).
+
+
```

Comment:
Adding a preamble about custom functions which can be used to achieve certain tasks might help users have some context on what these functions are for.